### DISCUSSION TITLE
5.1 under Windows 10 x64: window opens but is not visible

### DISCUSSION THREAD

**Discussion by DangerousPrototypes** [gusmanb/logicanalyzer#139] [General]
Hello,

I didn't want to start an issue because I assume this is a problem on my end. 

Using the latest release 5.1 for win x64 on Windows 10 downloaded from github. It launches with the logo, but then there appears to be a dialog behind it momentarily before it all disappears. 

![la-alt-tab](https://github.com/user-attachments/assets/e02cfef4-6270-403c-9ea7-f0783b511970)

The application icon is in the task bar, and I can ALT+TAB and see an open window that looks like this. I cannot get it to come onto the screen with Cascade Windows, etc.

I downloaded and install visual studio desktop runtime 7 because I saw it mentioned here, but that didn't seem to help.

I feel like I must be missing a a library or something. Any suggestions?

Thank you and best regards,

ian

**Comment by gusmanb** [comment]
Hmmm, I would say that the window settings have got corrupted somehow.

Try to delete the file "%AppData%\LogicAnalyzer\AppConfig.json" and check if this solves the problem.

Cheers.

**Reply by DangerousPrototypes** [reply]
Thank you so much! That fixed it.

Great work on the software, it is looking so good. Very excited for the future with the sigrok analyzer plugins.

I'm sure there's a nasty technical reason, but I do miss pan and zoom with mouse.

Thanks again, that got it going!

Ian

**Reply by gusmanb** [reply]
Glad it solved the problem 😄 

V6.0 allows to pan and zoom using the scroll wheel 😉 

Cheers.

**Reply by DangerousPrototypes** [reply]
I'm even more excited for V6 than before :)

**Reply by DeeJay** [reply]
_"V6.0 allows to pan and zoom using the scroll wheel "_

That's SO much more usable than the previous sliders!

V6.0 is going to be a big step up!
